modules/services/yggdrasil.nix

```diff
@@ -0,0 +1,117 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.yggdrasil;
+
+  settingsProvided = cfg.settings != { };
+  configFileProvided = cfg.configFile != null;
+
+  format = pkgs.formats.json { };
+in
+{
```

Comment:
Can you add yourself as a maintainer for this module?

Reply:
I did not contributed to nixpkgs yet, so I have no `lib.maintainers` record, so I leaved simple string with my username, is this okay?

Reply:
Can you change it to `maintainers.rubikoid or "rubikoid"` so if you ever end up maintaining packages in Nixpkgs it'll still refer to you